ProjectSource/src/main/java/com/kayako/sdk/base/requester/GetListRequester.java

```diff
@@ -0,0 +1,57 @@
+package com.kayako.sdk.base.requester;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class GetListRequester extends GetRequestProperty {
+
+  private final String endpoint;
+  private final String argId;
+
+  private static final String ARG_OFFSET = "offset";
+  private static final String ARG_LIMIT = "limit";
+
+  private long mId;
+  private String mHelpCenterUrl;
+  private Map<String, String> mQueryParams;
+
+  public GetListRequester(String endpoint, String argId, String helpCenterUrl, long id, int offset,
+      int limit) {
+
+    this.endpoint = endpoint;
+    this.argId = argId;
+    mId = id;
+    mHelpCenterUrl = helpCenterUrl;
+    mQueryParams = new HashMap<String, String>();
+    mQueryParams.put(this.argId, String.valueOf(this.mId));
```

Comment:
⚠️  Using the `this.` operator is recommended only when there is a naming collision between a local variable and a field, when needing to access the field. Modern IDEs mark local variables and fields differently, to avoid misleading developers into using the wrong reference. Thus it is recommended to avoid using `this.` unless it is unavoidable.  
          
## The solution in this case

```JAVA
mQueryParams.put(this.argId, String.valueOf(mId));
```
 

<a id="1b7e7fc0-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />